webapp/src/locales/fr.json

```diff
@@ -1,517 +1,518 @@
-{

```

Comment:
This file was encoded with CR+LF encoding. I changed it to unix style as in en.json and de.json